lib/kflags/kconfig/config.go

```diff
@@ -292,7 +295,13 @@ func NewConfigAugmenterFromDNS(cs cache.Store, domain string, binary string, mod
 	dns := remote.NewDNS(domain, append([]remote.DNSModifier{remote.WithLogger(options.log)}, options.dnso...)...)
 	eps, errs := dns.GetEndpoints()
 	if len(eps) <= 0 {
-		return nil, multierror.NewOr(errs, fmt.Errorf("no endpoints for domain '%s' could be detected - configure TXT records for %s? No connectivity?", domain, dns.Name()))
+		// try fallback endpoint
+		var ep remote.Endpoint
+		_, err := fallback.Unmarshal(endpoint_key, &ep)
```

Comment:
rather than use a constant, use the domain name as key? + some arbitrary namespace qualifier? like "config-discovery-" + domain?

If you use the Cache, it will hash your key, the important part is that it is unique, it's ok if it contains invalid / dangerous characters.

Reply:
Good idea, will push an update shortly.

Reply:
If we continue to use the config store, then the domain would already be scoped to the "fallback" namespace.